plugins/rbac-backend/src/service/role-manager.test.ts

```diff
@@ -270,7 +258,7 @@ describe('BackstageRoleManager', () => {
       expect(result).toBeFalsy();
     });
 
-    // user:default/mike should inherits from group:default/team-a
+    // user:default/mike should inherits from group:default/team-a x
```

Comment:
I think this is small typo:
```suggestion
    // user:default/mike should inherits from group:default/team-a
```